api/src/main/java/org/eclipse/microprofile/concurrent/ManagedExecutorConfig.java

```diff
@@ -64,9 +64,10 @@
      * inclusion of the prerequisites, even if not explicitly specified.</p>
      *
      * <p>A <code>ManagedExecutor</code> must fail to inject, raising
-     * <code>DefinitionException</code> on application startup, if the same
-     * context type is implicitly or explicitly included in this set
-     * as well as in the set specified by {@link #propagated}.</p>
+     * <code>DefinitionException</code> on application startup,
```

Comment:
By `DefinitionException` I suppose you mean [the one from CDI](https://github.com/cdi-spec/cdi/blob/master/api/src/main/java/javax/enterprise/inject/spi/DefinitionException.java), right? If so, might be worth using a FQCN but I know this is just nitpicking really :)

Reply:
Good idea, it's best to be precise here.  I added a commit that that includes the fully qualified class name both here as well as for all mentions of DeploymentException where we were also lacking the package name.